src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    static final int iDateStr = 0;
+    static final int iEmployeeName = 1;
+    static final int iHoursWorked = 2;
+    static final int intIncomePerHour = 3;
```

Comment:
Constants should have special naming convetion, and please don`t forger about access modifier